### PR TITLE
🔧 Update BIGMAP sections for AiiDAlab

### PR DESCRIPTION
**Pull request by mbercx**
Update of the sections of the BIGMAP tutorial for the AiiDAlab cluster.

#### Notes

* Removes the paragraphs on exporting the structure since we don't have X-forwarding available here. Structure can be visualized via REST API.
* Section on REST API still needs to be updated, see #279.

#### TODO
- [X] Basics section
- [x] Quantum ESPRESSO section
- [x] Querying section

### CODE REVIEW

**Review by ramirezfranciscof — CHANGES_REQUESTED** [review]
In general ok, some comments to discuss/change and then I can approve.

Two more general comments/reminders:
- This contains a query section already incorporated; we will check this in #282 and then incorporate the changes in this branch as well
- The last QE workchain seems currently broken as indicated in issue #285. In the PR that fixes that we should check the last section on creating the band structure.

Also: this wasn't modified in this PR (so I can't add the comment on the code) but I just noticed that [in this part](https://github.com/aiidateam/aiida-tutorials/pull/284/files#diff-4619dfe538ceb00f99bc55b5fa9d0763929f7a6644c0eb019c268c0782865201R506-R519), it might be better to add some small comment like "remember to change the `<PK>` for that of the output of the mulitply calcfunction".

File: docs/pages/2020_BIGMAP/sections/qe.rst [hunk 1]
```diff
@@ -62,30 +62,15 @@ Let us first inspect the node you just created:
...
-    $ verdi data structure export --format=xsf <PK> > exported.xsf
-
-You can copy the file to your local machine and open it with `XCrySDen`_, or simply have a look at the structure using ASE's `visualization tools`_ with:
-
-.. code-block:: console
-
-    $ verdi data structure show <PK>
-
-This can take some time due to the X forwarding, but after a while you should be able to see the Si supercell (8 atoms) that we downloaded from the COD database (in CIF format), imported into AiiDA and exported back into a different format (XSF).
-
```
[the remark this answers is not in the document]
Comment (mbercx) [reply docs/pages/2020_BIGMAP/sections/qe.rst:88]: See comment in OP. We don't have X-forwarding on the AiiDAlab cluster. Maybe we can also visualize the structure via the AiiDAlab interface  (@csadorf ?), but I suggested using the REST API later.

File: docs/pages/2020_BIGMAP/sections/qe.rst [hunk 2]
```diff
@@ -62,30 +62,15 @@ Let us first inspect the node you just created:
...
 Running a calculation
```
[the remark this answers is not in the document]
Comment (mbercx) [reply docs/pages/2020_BIGMAP/sections/qe.rst:74]: Good point! 👍  This is normally explained in the setup section, but for this tutorial it's better just to add an example here. (or indeed maybe just the command, once the branch is in master)

File: docs/pages/2020_BIGMAP/sections/basics.rst [hunk 3]
```diff
@@ -430,16 +415,18 @@ Now, exit the IPython shell and once more check for *all* processes:
 .. code-block:: console
 
     $ verdi process list -a
-      PK  Created    Process label             Process State    Process status
-    ----  ---------  ------------------------  ---------------  ----------------
-     185  1h ago     multiply                  ⏹ Finished [0]
-     189  4m ago     ArithmeticAddCalculation  ⏹ Finished [0]
+    PK  Created    Process label                 Process State    Process status
+    ----  ---------  ----------------------------  ---------------  ----------------
+    <! OUTPUT REMOVED !>
```
[the remark this answers is not in the document]
Comment (mbercx) [reply docs/pages/2020_BIGMAP/sections/basics.rst:420]: There is a note just after this output:

```
Note that we've removed the output regarding the band structure calculation that you ran in the `AiiDAlab`_ `Quantum ESPRESSO`_ app earlier.
```

 Isn't that sufficient?

File: docs/pages/2020_BIGMAP/sections/basics.rst [hunk 4]
```diff
@@ -641,18 +627,19 @@ Depending on which step the workflow is running, you should get something like t
...
-     204  5s ago     ArithmeticAddCalculation  ⏵ Waiting        Monitoring scheduler: job state RUNNING
+      PK  Created    Process label                 Process State    Process status
+    ----  ---------  ----------------------------  ---------------  ------------------------------------
+    <! OUTPUT REMOVED !>
+     148  18m ago    multiply                      ⏹ Finished [0]
+     152  3m ago     ArithmeticAddCalculation      ⏹ Finished [0]
+     157  1m ago     ArithmeticAddCalculation      ⏹ Finished [0]
+     164  4s ago     MultiplyAddWorkChain          ⏵ Waiting        Waiting for childprocesses: 167
+     165  3s ago     multiply                      ⏹ Finished [0]
+     167  3s ago     ArithmeticAddCalculation      ⏵ Waiting        Waiting for transport task: retrieve
```
[the remark this answers is not in the document]
Comment (mbercx) [reply docs/pages/2020_BIGMAP/sections/basics.rst:631]: Hehe, yeah, you have to be very quick (originally the `safe_interval` for the `localhost` wasn't zero). Maybe I can add a note here and show it during the tutorial. 👍

**Comment by mbercx** [comment]
Thanks for the review, @ramirezfranciscof! I left my comments above.

File: docs/pages/2020_BIGMAP/sections/basics.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (mbercx) [reply docs/pages/2020_BIGMAP/sections/basics.rst:631]: I'll just add `!verdi process list -a` 🙃

```Python
In [3]: from aiida.engine import submit
   ...: submit(builder)
   ...: !verdi process list -a
```

File: docs/pages/2020_BIGMAP/sections/basics.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (ramirezfranciscof) [reply docs/pages/2020_BIGMAP/sections/basics.rst:420]: Haha, sorry, I didn't see that. Yes, that should be ok.

File: docs/pages/2020_BIGMAP/sections/qe.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (ramirezfranciscof) [reply docs/pages/2020_BIGMAP/sections/qe.rst:88]: Ah, I see. Ok, I'll try to remember it for the restapi part (anyways my idea was start from a band structure and trace back to see the original structure, so it works I guess).

File: docs/pages/2020_BIGMAP/sections/basics.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (ramirezfranciscof) [reply docs/pages/2020_BIGMAP/sections/basics.rst:631]: Haha, that could be a solution, but we would need to explain what the `!` does maybe.

**Review by ramirezfranciscof — APPROVED** [review]
Saul goodman!

**Comment by mbercx** [comment]
> Saul goodman!

saul

**Pull request merged**